modules/mobile/spec/request/facilities_info_request_spec.rb

```diff
@@ -105,8 +105,8 @@
             get '/mobile/v0/facilities-info/home', headers: iam_headers, params: params
             facilities = response.parsed_body.dig('data', 'attributes', 'facilities')
             expect(response).to have_http_status(:ok)
-            expect(facilities[0]['id']).to eq('442')
```

Comment:
FYI, the description of this test no longer makes sense. The user starts with the ids 983/984 and that's what they still are, so there's no conversion. The same thing applies to the test below.

Reply:
good catch. Removed the unused specs. not going to worry about force, converting to 9xx ids since the test users profiles ids are already 9xx